src/renderers/dom/shared/DOMMarkupOperations.js

```diff
@@ -99,8 +99,9 @@ var DOMMarkupOperations = {
         (propertyInfo.hasOverloadedBooleanValue && value === true)
       ) {
         return attributeName + '=""';
+      } else if (typeof value !== 'boolean' || DOMProperty.allowBoolean(name)) {
+        return attributeName + '=' + quoteAttributeValueForBrowser(value);
       }
-      return attributeName + '=' + quoteAttributeValueForBrowser(value);
```

Comment:
I'd love to just lean on `DOMProperty.shouldSetAttribute`, but reserved props pass through here, and style is in there. Maybe we could remove `style` from reserved props.